src/app/games/siam/SiamFailure.ts

```diff
@@ -1,10 +1,10 @@
 export abstract class SiamFailure {
 
-    public static readonly NO_REMAINING_PIECE_TO_INSERT: string = $localize`Vous ne pouvez plus insérer, toutes vos pièces sont déjà sur le plateau!`;
+    public static readonly NO_REMAINING_PIECE_TO_INSERT: string = $localize`You cannot insert a piece, all your pieces are already on the board.`;
 
-    public static readonly NOT_ENOUGH_FORCE_TO_PUSH: string = $localize`Vous ne pouvez pas pousser, vous n'avez pas assez de forces`;
+    public static readonly NOT_ENOUGH_FORCE_TO_PUSH: string = $localize`You do not have enough strength to push.`;
 
-    public static readonly ILLEGAL_ROTATION: string = $localize`Vous ne pouvez pas tourner et avancer en même temps!`;
+    public static readonly ILLEGAL_ROTATION: string = $localize`You cannot turn and move forward at the same time.`;
 
```

Comment:
For both language: this is a false sentence: you cannot PUSH and turn at the same time !